plutarch-test/plutarch-base/Plutarch/ShowSpec.hs

```diff
@@ -11,69 +11,71 @@ import Test.Hspec
 
 spec :: Spec
 spec = do
-  describe "show" . pgoldenSpec $ do
```

Comment:
the only thing that was changed here is that the `let str x = pconstant @PString x` got hoisted upwards and `describe "show"` now houses 2 separate `pgoldenSpec`s, one has `plutarchDevFlagDescribe`